src/index.pug

```diff
@@ -3,7 +3,7 @@ html
   head
     meta(charset='utf-8')
     meta(name='viewport', content='width=device-width, initial-scale=1, shrink-to-fit=no')
-    link(rel='stylesheet', href='//code.cdn.mozilla.net/fonts/fira.css')
+    link(rel='stylesheet', href='//code.cdn.mozilla.net/fonts/zilla-slab.css')
     link(rel='stylesheet', href='css/mofo-bootstrap.css')
```

Comment:
`css/mofo-bootstrap.css` was a 404 so I made changes to `package.json` to copy this file over to `demo/css/`